components/stepper/index.css

```
@@ -315,6 +295,15 @@ governing permissions and limitations under the License.
     border-radius: 0;
     inline-size: var(--mod-stepper-quiet-width, var(--spectrum-stepper-quiet-width));
 
+    &::after {
+      content: '';
+      position: absolute;
+      bottom: calc(-1 * var(--mod-stepper-focus-indicator-gap, var(--spectrum-stepper-focus-indicator-gap)) * 2);
```

I think this should be...
```suggestion
      bottom: calc(-1 * (var(--mod-stepper-focus-indicator-gap, var(--spectrum-stepper-focus-indicator-gap)) + var(--mod-textfield-focus-indicator-width, var(--spectrum-textfield-focus-indicator-width)));
```
(Mod the focus-indicator-width to a bigger value and you'll see; this should still maintain the gap).